packages/browser/types/test/fixtures/app.js

```diff
@@ -1,10 +0,0 @@
-"use strict";
-exports.__esModule = true;
-var bugsnagInstance = undefined;
-function notify(error, opts) {
-    if (bugsnagInstance === undefined) {
-        return false;
-    }
-    return bugsnagInstance.notify(error, opts);
-}
-exports.notify = notify;
```

Comment:
Are these two files no longer used?

Reply:
They get generated by the tests and were erroneously checked in.